node-manager/src/node.rs

```diff
@@ -28,13 +29,135 @@ impl Node {
         let keys_manager = create_keys_manager(mnemonic, node_index.child_index);
         let pubkey = pubkey_from_keys_manager(&keys_manager);
 
-        let tcp_proxy = Arc::new(TcpProxy::new().await);
-
         Ok(Node {
             uuid: node_index.uuid,
             pubkey,
             keys_manager: Arc::new(keys_manager),
-            tcp_proxy,
         })
     }
+
+    pub async fn connect_peer(
+        &self,
+        websocket_proxy_addr: String,
+        peer_pubkey_and_ip_addr: String,
+    ) -> Result<(), Box<dyn std::error::Error>> {
+        if peer_pubkey_and_ip_addr.is_empty() {
+            info!(
```

Comment:
this should be an error log

Reply:
Moved all of these logs out of here and handling at the caller.